Hello_Object_Oriented_Due_Dec_1/renderBoard.js

```diff
@@ -0,0 +1,132 @@
+
+
+class Task {
+  constructor(taskName='', assignee='') {
+    this.taskName = taskName;
+    this.assignee = assignee;
+  }
+
+  render() {
+    if (this.assignee != '' && this.taskName != '') {
+
+      return `${this.taskName} ${this.assignee}`;
+    }
+    else if (this.taskName != '') {
+
+      return `${this.taskName}`;
+    } else {
+      return '';
+    }
+  }
+}
+
+
+class List {
+
+  constructor(listName) {
+    this.listName = listName;
+    this.lists = []; // store tasks
```

Comment:
you should names this `this.tasks` since it is a list of tasks, it is not a list of lists. Naming is `this.tasks` would be less confusing